cob_sound/ros/src/sound.cpp

```diff
@@ -130,7 +163,45 @@ class SayAction
     return true;
   }
   
-  
+  bool play(std::string filename)
+  {
+    if (mute_)
+    {
+      ROS_WARN("Sound is set to mute. You will hear nothing.");
+      return true;
+    }
+    if(!nh_.hasParam("audio_file_path"))
+    {
+      ROS_ERROR("Parameter 'audio_file_path' not set");
+      return false;
+    }
+    std::string command;
+    std::string audio_file_path;
+    nh_.getParam("audio_file_path",audio_file_path);
+    std::string audio_file= audio_file_path +"/"+filename+".wav";
```

Comment:
Few comments here:
 - Why do we check for parameter existence every `play()`. Can we not have this in the constructor/init as a member variable?
 - I'd better like if we could specify `audio_file_path` in a `${find package_name}` fasion
 - Can we only play `*.wav`? I'd prefer to not have the file-suffix added hardcoded like this. I'd rather expect the user to fully name the filename __with__ suffix. If we can only play `*.wav`'s we could reject and return `false`.